src/main/scala/de/upb/cs/swt/delphi/instanceregistry/io/swagger/client/model/Event.scala

```diff
@@ -79,9 +80,21 @@ trait EventJsonSupport extends SprayJsonSupport with DefaultJsonProtocol with In
     }
 
   }
+  implicit val timestamp: JsonFormat[DateTime] = new JsonFormat[DateTime] {
+    override def write(obj: DateTime) = JsString(obj.toIsoDateTimeString())
+
+    override def read(json: JsValue): DateTime = json match {
+      case JsString(value) =>
+        DateTime.fromIsoDateTimeString(value) match {
+          case Some(date) => date
+          case _ => throw new DeserializationException("Failed to parse date time [" + value + "].")
+        }
+      case _ => throw new DeserializationException("Failed to parse json string [" + json + "].")
```

Comment:
Please remove the ```new``` keyword here, it is not necessary and marked as warning by IntelliJ.